src/collections/config/types/vectorizer.ts

```diff
@@ -393,8 +393,6 @@ export type Text2MultiVecJinaAIConfig = {
   dimensions?: number;
   /** The model to use. */
   model?: string;
-  /** Whether to vectorize the collection name. */
-  vectorizeCollectionName?: boolean;
```

Comment:
This type has been first introduced in 3.7.0. As we're deprecating that version entirely, might as well delete the unwanted config parameter.

Reply:
Is it unwanted? Are there any cases someone might want this ?

Reply:
We're doing this as part of an effort to improve the UX around vectorization. Long story short setting this to `true` can get Weaviate to produce useless vectors for collections that actually have no embeddable properties.

Instead of returning an error on write, we would just generate an embedding of the collection name every time. 

It is also meaningless to some of the modules, like `text2multivec-jinaai` in the example above, but it made its way to the API cuz it seemed plausible :)